docs/6-external-interface/4-guardians-registry.md

```diff
@@ -173,17 +173,6 @@ The following functions are state getters provided by the `GuardiansRegistry`:
     - **Locked:** Amount of active tokens that are locked due to ongoing disputes
     - **Pending deactivation:** Amount of active tokens that were requested for deactivation
 
-#### 6.4.2.7. Detailed balance of at
```

Comment:
I know it's a PITA, but should we update the following indexes?

Reply:
Ah yes totally, forgot about it!